oracle/price-feeder/README.md

```diff
@@ -9,6 +9,66 @@ This is a standalone version of [Umee's fantastic work](https://github.com/umee-
 
 ---
 
+
+# Setup
+If a cluster is running Oracle price-feeder, your validator is also required to run a price feeder or your validator will be jailed for missing votes.
+
+
+## Create an account for Oracle Price Feeder Delegate
+1) To avoid account sequence errors, it's reccomended to create a different account as an Oracle delegate. To do so, you'll need to create the account with
+`seid keys add price-feeder-delegate` or any other account name.
+2) With the account address output, `export PRICE_FEEDER_DELEGATE_ADDR=<output>`
+3) `seid tx oracle set-feeder {feeder-bech32} --from $PRICE_FEEDER_DELEGATE_ADDR --fees 2000usei -b block -y --chain-id {chain-id}`
```

Comment:
I believe this is in error.  Should be:

`seid tx oracle set-feeder $PRICE_FEEDER_DELEGATE_ADDR --from <validator-wallet> --fees 2000usei -b block -y --chain-id {chain-id}`

Reply:
ah yes, thank you!